packages/levenshtein/levenshtein.1.1.3/descr

```diff
@@ -0,0 +1,2 @@
+Levenshtein distance algorithm for general array.
+Levenshtein distance algorithm for general array.
```

Comment:
I think it's recommended to have a new line between the short and long descriptions